### ISSUE TITLE
GamepadHapticActuator needs to define the situation of loss focus

### ISSUE THREAD

**Issue by daoshengmu** [w3c/gamepad#58]
In the spec of [pulse](https://w3c.github.io/gamepad/extensions.html#dom-gamepadhapticactuator-pulse()), we have already defined how to submit a duration for a pulse vibration. But, we didn't consider when we submit a very long pulse vibration, then we leave our focus to another tab. We should let the controller keep vibrating? Or we should stop it when we loss focus.

@toji , I think you might have some thought about this.

**Comment by marcoscaceres** [comment]
This is related to "fully active"... so definitely something we should define. 

This is in the extensions document, so it's not actually part of the spec?

**Comment by daoshengmu** [comment]
> This is related to "fully active"... so definitely something we should define.
> 
> This is in the extensions document, so it's not actually part of the spec?

Yep. this is in the extensions doc and seems like we already merge extensions doc under this repo.
